galaxy/values.yaml

```diff
@@ -349,11 +380,50 @@ configs:
       file: "/galaxy/server/tool-data/shared/igv/igv_build_sites.txt.sample"
     - type: rviewer
       file: "/galaxy/server/tool-data/shared/rviewer/rviewer_build_sites.txt.sample"
+  # Although this is only one line, the multi-line entry allows us to avoid wrapping
+  # the entire expression in quotes, which would need to be removed in the configmap
+  integrated_tool_panel.xml: |
+    {{- (.Files.Get "files/configs/integrated_tool_panel.xml") }}
   tool_conf.xml: |
     <?xml version='1.0' encoding='utf-8'?>
     <toolbox monitor="true">
       <section id="get_data" name="Get Data">
         <tool file="data_source/upload.xml" />
+        <tool file="data_source/ucsc_tablebrowser.xml" />
+        <tool file="data_source/ucsc_tablebrowser_archaea.xml" />
+        <tool file="data_source/ebi_sra.xml" />
+        <tool file="data_source/fly_modencode.xml" />
+        <tool file="data_source/intermine.xml" />
+        <tool file="data_source/flymine.xml" />
+        <tool file="data_source/modmine.xml" />
+        <tool file="data_source/mousemine.xml" />
+        <tool file="data_source/ratmine.xml" />
+        <tool file="data_source/yeastmine.xml" />
+        <tool file="data_source/worm_modencode.xml" />
+        <tool file="data_source/wormbase.xml" />
+        <tool file="data_source/zebrafishmine.xml" />
+        <tool file="data_source/eupathdb.xml" />
+        <tool file="data_source/hbvar.xml" />
+        <tool file="genomespace/genomespace_importer.xml" />
+        <tool file="genomespace/genomespace_push.xml" />
```

Comment:
Do all these work? I think it's pretty safe to omit genomespace entries at least.

Reply:
I'm not sure, this is from the previous `values-cvmfs`. It doesn't break startup for either case but I have no idea if they work in either.

Reply:
I think this should probably be a separate file anyway imo